integration-tests/utils/integration-utils.js

```diff
@@ -160,8 +160,9 @@ const select = async (page, labelContains, text) => {
   return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
 }
 
-const waitForNoSpinners = page => {
-  return page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
+const waitForNoSpinners = async page => {
+  await page.waitForXPath('//*[@id="root"]', { timeout: 60 * 1000 })
```

Comment:
All Terra UI pages have this root element.

Reply:
What case are you trying to guard against? This will make sure that there is a `root` element, but that only guarantees that the original `index.html` is loaded, not necessarily that the app JS is loaded.

Reply:
Intention is to prevent checking spinner too early. e.g. page is a blank, or if ui failed to load. Can you think a better way to prevent checking spinner too early?